forge-gui/res/cardsfolder/upcoming/ajani_nacatl_pariah_ajani_nacatl_avenger.txt

```diff
@@ -0,0 +1,35 @@
+Name:Ajani, Nacatl Pariah
+ManaCost:1 W
+Types:Legendary Creature Cat Warrior
+PT:1/2
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When CARDNAME enters the battlefield, create a 2/1 white Cat Warrior creature token.
+SVar:TrigToken:DB$ Token | TokenAmount$ 1 | TokenScript$ w_2_1_cat_warrior | TokenOwner$ You
+T:Mode$ ChangesZoneAll | TriggerZones$ Battlefield | ValidCards$ Cat.Other+YouCtrl | Origin$ Battlefield | Destination$ Graveyard | Execute$ TrigExile | OptionalDecider$ You | TriggerDescription$ Whenever one or more other Cats you control die, you may exile NICKNAME, then return him to the battlefield transformed under his owner's control.
+SVar:TrigExile:DB$ ChangeZone | Origin$ Battlefield | Destination$ Exile | RememberChanged$ True | SubAbility$ DBReturn
+SVar:DBReturn:DB$ ChangeZone | Defined$ Remembered | Origin$ Exile | Destination$ Battlefield | Transformed$ True | ForgetOtherRemembered$ True | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+DeckHas:Ability$Token
+AlternateMode:DoubleFaced
+Oracle:When Ajani, Nacatl Pariah enters the battlefield, create a 2/1 white Cat Warrior creature token.\nWhenever one or more other Cats you control die, you may exile Ajani, then return him to the battlefield transformed under his owner's control.
+
+ALTERNATE
+
+Name:Ajani, Nacatl Avenger
+ManaCost:no cost
+Colors:red,white
+Types:Legendary Planeswalker Ajani
+Loyalty:3
+A:AB$ PutCounterAll | Cost$ AddCounter<2/LOYALTY> | ValidCards$ Cat.YouCtrl | CounterType$ P1P1 | CounterNum$ 1 | Planeswalker$ True | SpellDescription$ Put a +1/+1 counter on each Cat you control.
+A:AB$ Token | Cost$ AddCounter<0/LOYALTY> | TokenAmount$ 1 | TokenScript$ w_2_1_cat_warrior | TokenOwner$ You | RememberTokens$ True | SubAbility$ DBImmediateTrig1 | Planeswalker$ True | SpellDescription$ Create a 2/1 white Cat Warrior creature token. When you do, if you control a red permanent other than CARDNAME, he deals damage equal to the number of creatures you control to any target.
+SVar:DBImmediateTrig1:DB$ ImmediateTrigger | ConditionDefined$ Remembered | ConditionPresent$ Card | Execute$ ImmediateTrig2 | SubAbility$ DBCleanup | TriggerDescription$ When you do,...
+SVar:ImmediateTrig2:DB$ ImmediateTrigger | ConditionPresent$ Permanent.Red+YouCtrl+Other | Execute$ TrigDamage | TriggerDescription$ When you do, if you control a red permanent other than CARDNAME, he deals damage equal to the number of creatures you control to any target.
```

Comment:
this would give you two reflexive triggers
you should only use one, ConditionPresent2$ exists so you can have both
and the DealDamage needs to check for it too

Reply:
Much obliged for the pointer :) Done and tested. Unfortunately this pull request even split is one big mess so I'll be starting this one over on a fresh branch. My apologies for that.